qi-test/tests/compiler/rules/full-cycle.rkt

```diff
@@ -39,9 +45,8 @@
     (test-true "normalize → deforest"
                (deforested?
                  (phase1-eval
-                  (test-compile~> #'(~>> (filter odd?) values (map sqr))
-                                  normalize-pass
-                                  deforest-pass)))))))
+                  (qi-compile
+                   #'(~>> (filter odd?) values (map sqr)))))))))
```

Comment:
I think the test became more implicit here, right? `qi-compile` is expected to both normalize and deforest (through the composition of `compile-flow` and `expand-flow`); in the old version, using both passes was explicit.

Reply:
That explicit composition could be useful and that's why I initially left the macro in there though it was unused. But this module is called "full-cycle" so it's intended to treat the entire compilation as a single operation. For now I've removed the unused macro but if we feel it's useful we could introduce a separate module to test the impact of specific sequences of passes.